core/src/instance.rs

```diff
@@ -86,11 +86,18 @@ impl Instance {
         (rx_action, rx_observer)
     }
 
+    pub fn initialize_context(&self, context: Arc<Context>) -> Arc<Context> {
```

Comment:
`pub` doesn't seem necessary.

Reply:
In my understanding of Rust, tests in same crate don't care about visibility, so no need for `pub` for tests.

Reply:
But all tests that create an instance and process actions manually (or call reducer functions, etc.) without running the redux loop the regular why need to construct a context and thus need to call this function. Take a look at #315.